frontend/src/lib/components/PropertyFilters/PropertyValue.js

```diff
@@ -5,6 +5,7 @@ import { isMobile, isOperatorFlag, isOperatorMulti, isOperatorRegex, isValidRege
 import { SelectGradientOverflow } from 'lib/components/SelectGradientOverflow'
 
 export function PropertyValue({
+    // TODO: Convert to TS.
```

Comment:
Note - no real reason to add this TODO as we'll slowly convert all .js files over.